infra/aws/terraform/modules/eks-prow-iam/policies.tf

```
@@ -14,9 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+resource "aws_iam_policy" "eks_plan" {
+  path   = "/"
+  policy = data.aws_iam_policy_document.eks_plan.json
+  tags   = var.tags
+}
+
 data "aws_iam_policy_document" "eks_plan" {
   statement {
-    sid       = ""
+    sid       = "EKSRead"
```

```suggestion
    sid       = "AllowReadEKSSupportingInfra"
```
_and_
move `sts:AssumeRole` into a separate statement - it's not a read operation.